bin/picca_fast_metal_dmat.py

```diff
@@ -0,0 +1,570 @@
+            r_trans_all.append(r_trans_eff)
+            z_all.append(z_eff)
+            names.append(abs_igm1 + "_" + abs_igm2)
+            #weights_dmat_all.append(weights_dmat)
+            #r_trans_all.append(r_trans) NA
+            #num_pairs_all.append(num_pairs) NA
+            #num_pairs_used_all.append(num_pairs_used) NA
+
+    t2 = time.time()
+    userprint(f'picca_fast_metal_dmat.py - Time computing all metal matrices : {(t2-t1)/60:.3f} minutes')
+
+    # save the results
+    results = fitsio.FITS(args.out, 'rw', clobber=True)
+    header = [
+        {
+            'name': 'RPMIN',
+            'value': cf.r_par_min,
+            'comment': 'Minimum r-parallel [h^-1 Mpc]'
+        },
+        {
+            'name': 'RPMAX',
+            'value': cf.r_par_max,
+            'comment': 'Maximum r-parallel [h^-1 Mpc]'
+        },
+        {
+            'name': 'RTMAX',
+            'value': cf.r_trans_max,
+            'comment': 'Maximum r-transverse [h^-1 Mpc]'
+        },
+        {
+            'name': 'NP',
+            'value': cf.num_bins_r_par,
+            'comment': 'Number of bins in r-parallel'
+        },
+        {
+            'name': 'NT',
+            'value': cf.num_bins_r_trans,
+            'comment': ' Number of bins in r-transverse'
+        },
+        {
+            'name': 'COEFMOD',
+            'value': args.coef_binning_model,
+            'comment': 'Coefficient for model binning'
+        },
+        {
+            'name': 'ZCUTMIN',
+            'value': cf.z_cut_min,
+            'comment': 'Minimum redshift of pairs'
+        },
+        {
+            'name': 'ZCUTMAX',
+            'value': cf.z_cut_max,
+            'comment': 'Maximum redshift of pairs'
+        },
+        {
+            'name': 'REJ',
+            'value': cf.reject,
+            'comment': 'Rejection factor'
+        },
+        {
+            'name': 'ALPHAMET',
+            'value': args.metal_alpha,
+            'comment': 'Evolution of metal bias'
+        }, {
+            'name': 'OMEGAM',
+            'value': args.fid_Om,
+            'comment': 'Omega_matter(z=0) of fiducial LambdaCDM cosmology'
+        }, {
+            'name': 'OMEGAR',
+            'value': args.fid_Or,
+            'comment': 'Omega_radiation(z=0) of fiducial LambdaCDM cosmology'
+        }, {
+            'name': 'OMEGAK',
+            'value': args.fid_Ok,
+            'comment': 'Omega_k(z=0) of fiducial LambdaCDM cosmology'
+        }, {
+            'name': 'WL',
+            'value': args.fid_wl,
+            'comment': 'Equation of state of dark energy of fiducial LambdaCDM cosmology'
+        }, {
+            'name': "BLINDING",
+            'value': blinding,
+            'comment': 'String specifying the blinding strategy'
+        }
+        ]
+    len_names = np.array([len(name) for name in names]).max()
+    names = np.array(names, dtype='S' + str(len_names))
+    results.write(
+        [
+            #np.array(num_pairs_all),
```

Comment:
remove commented lines (I take it they were used for code development)

Reply:
done